src/main/java/frc/robot/subsystems/manipulator/ShooterSubsystem.java

```diff
@@ -15,7 +15,7 @@
 import java.util.function.DoubleSupplier;
 
 public class ShooterSubsystem extends SubsystemBase {
-  private final boolean SHUFFLEBOARD_ENABLED = false;
+  private final boolean SHUFFLEBOARD_ENABLED = true;
```

Comment:
This can be false.

Reply:
OK, I was just trying to get the telemetry to monitor it.